### DISCUSSION TITLE
使用时dwm.exe显卡占用非常高

### DISCUSSION THREAD

**Discussion by longisland-icetea** [Richasy/Bili.Uwp#658] [问答]
使用时dwm.exe显卡占用非常高，随便点点就能占用HD630的70%以上，这是正常的吗？

**Comment by hez2010** [comment]
正常，因为dwm负责整个系统的图形加速窗口绘制，哔哩UWP含有大量UI元素需要绘制自然会导致dwm GPU占用上升。
